src/python/impactx/dashboard/Toolbar/exportTemplate.py

```diff
@@ -0,0 +1,72 @@
+import re
+
+from trame.app import get_server
+
+# -----------------------------------------------------------------------------
+# Trame setup
+# -----------------------------------------------------------------------------
+
+server = get_server(client_type="vue2")
+state, ctrl = server.state, server.controller
+
+# -----------------------------------------------------------------------------
+# Content
+# -----------------------------------------------------------------------------
+
+
+def retrieve_state_content():
+    """
+    This function builds the output file based on user inputs

```

Comment:
You mean the input file?
```suggestion
    This function builds the input file based on user inputs

```

Reply:
Since this function is for exporting user inputs,  should it be "builds the 'export'" instead? When I think of input file, I am thinking of a file the user would provide to the system, but this function is meant for the other way around.

Reply:
Yes, that is more explicit. Please change as you suggested :)